appserver/extras/docker-images/server-full/src/main/docker/Dockerfile

```diff
@@ -29,25 +29,25 @@ ENV PATH="${PATH}:${PAYARA_DIR}/bin"
 ARG TINI_VERSION=v0.18.0
 
 # Download tini
-ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini \
-    https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini.asc /
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini.asc /tini.asc
 
 RUN true \
     # Install dependencies
     && apt-get update \
-    && apt-get install -y coreutils gpg \
+    && apt-get install -y gpg \
     && rm -rf /var/lib/apt/lists/* \
     # Create and set the Payara user and working directory owned by the new user
-    && groupadd --gid 1000 payara \
-    && useradd -u 1000 -M -s /bin/bash -d ${HOME_DIR} payara -g payara \
+    && addgroup --gid 1000 payara \
+    && adduser --system --uid 1000 --no-create-home --shell /bin/bash --home "${HOME_DIR}" --gecos "" --ingroup payara payara \
     && echo payara:payara | chpasswd \
     && mkdir -p ${DEPLOY_DIR} \
     && mkdir -p ${CONFIG_DIR} \
     && mkdir -p ${SCRIPT_DIR} \
     && chown -R payara: ${HOME_DIR} \
     # Verify tini
-    && gpg --batch --keyserver "hkp://ipv4.pool.sks-keyservers.net" --recv-keys 595E85A6B1B4779EA4DAAEC70B588DFF0527A9B7 \
-    && gpg --batch --verify /tini.asc /tini \
+    && gpg --verbose --keyserver @docker.keyserver.url@ --recv-keys 595E85A6B1B4779EA4DAAEC70B588DFF0527A9B7 \
```

Comment:
Since this is source of majority of build failures, I'd propose specific exit code (until we implement different issue for taking this away completely from main build). 
Like `(gpg --recvkeys... || exit 55)`

Reply:
We already discussed it - exit codes are not much helpful when you don't remember what they mean. So until somebody would provide such code (for cycle?), 
`--verbose` + `-Ddocker.verbose` is more helpful as it directly prints what it tried to do and why it failed.